src/app/pages/courses/components/find-course/find-course.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="search-container">
```

Comment:
lets avoid of additional wrappers creation - to style the whole component you could use :host pseudo class in css